README.md

```diff
@@ -48,10 +48,19 @@ If you prefer the command line way of downloading and installing things, then fe
   ##### `git clone https://github.com/lugenx/ecohabit.git`
 - Open the cloned folder in VS Code. Open the terminal and make sure its pointing to the root of the cloned project.
 
+
+- Database creation
+  - Create a mongodb account through the [MongoDB Website](https://www.mongodb.com)
+  - Create a cluster (by default is Cluster0 which can be used)
+  - Once the cluster is up, you need to click on the 'Connect' tab and follow the drivers section to retrieve the mongodb connection url and save it for the next steps
+
+<br>
+
 - Running Backend Node JS Application:
```

Comment:
can we please change this old text too?

"Run the backend NodeJS application:"